index.php

```diff
@@ -1,4 +1,8 @@
 <?php
+error_reporting(E_ALL);
```

Comment:
for reporting error, you should create a single file, as like i said above for DB connection.